### ISSUE TITLE
blacklisted domains

### ISSUE THREAD

**Issue by luismmontielg** [gggritso/Vimmy.safariextension#5]
I would like to disable the extension in some websites, like gmail.com

**Comment by gggritso** [comment]
Do you think it makes sense to also have a global hard-coded list of blacklisted domains? There are probably other sites that have their own keyboard navigation. GitHub is another example.
